src/views/Maintenance.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div>
+    <div class="maintenance container wrapper mt-2">
```

Comment:
could you please do not use mt-2, pt-2, text-center and other bootstrap classes (Bootstrap is useless, we would like to clean up and remove it). Instead add the properties to the specific class (add one if missing). In this case add margins to `.maintainance`. You can leave `.container`+`.wrapper` if needed.